### ISSUE TITLE
[CoE Starter Kit - QUESTION] QUESTION

### ISSUE THREAD

**Issue by ashlega** [microsoft/coe-starter-kit#4599]
### What is your question?

Hi,

  in one of our environments, we have a BPF which is owned by a team (I don't know how that happened - there are  lots of environments, all kinds of things are happening there).

  However, this flow seems to be expecting BPF owner to be a user, not a team:

  Admin | Sync Template v3 (Business Process Flows)

![image](https://user-images.githubusercontent.com/5911927/212732078-1895b0e7-f10e-413d-b210-7dba52e84d90.png)

  So it keeps failing for that particular BPF.
 
  I wonder if this would be something to fix or if this is expected behavior?

Thanks,
Alex

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.56

### What app or flow are you having the issue with?

Admin | Sync Template v3 (Business Process Flows)

**Comment by Jenefer-Monroe** [comment]
Is that the call its failing on?

**Comment by ashlega** [comment]
Yes, that's the one:

![image](https://user-images.githubusercontent.com/5911927/212765870-78ec5c20-c541-4f7c-9ad1-1989efcf5e9a.png)

**Comment by Jenefer-Monroe** [comment]
and that "user" (5bac864f.....) do they exist in the "Team" system table in that environment?

**Comment by ashlega** [comment]
Yep, I found it hard to believe a BPF would have been assigned to a team (it seems to be the only BPF assigned to a team in the whole tenant :)) And the only one I've ever seen assigned to a team). But, yes, the team exists:

![image](https://user-images.githubusercontent.com/5911927/212800934-e166396c-a6de-4e63-b8d3-7bef0d93008f.png)

**Comment by mohamrizwa** [comment]
We noticed that this issue has been pending for quite some time. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved.
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue.
Thank you for your understanding and cooperation.
